daprdocs/content/en/developing-applications/building-blocks/pubsub/pubsub-cloudevents.md

```diff
@@ -66,6 +66,45 @@ As another example of a v1.0 CloudEvent, the following shows data as XML content
 }
 ```
 
+### Override CloudEvent values
+
+You can override CloudEvent values by replacing some metadata with the following:
+- `cloudevent-id`: overrides `id`
+- `cloudevent-source`: overrides `source`
+- `cloudevent-type`: overrides `type`
+- `cloudevent-traceid`: overrides `traceid`
+- `cloudevent-tracestate`: overrides `tracestate`
+- `cloudevent-traceparent`: overrides `traceparent`
+
+{{% alert title="Note" color="primary" %}}
+The following metadata can't be overridden:
+- `data`
+- `topic`
+- `pubsub`
+- `datacontenttype`
+{{% /alert %}}
+
+For example, to override the applicable values from [the first CloudEvent example above]({{< ref "#cloudevents-example" >}}):
+
+```json
+{
+  "topic": "orders",
+  "pubsubname": "order_pub_sub",
+  "cloudevent-traceid": "00-113ad9c4e42b27583ae98ba698d54255-e3743e35ff56f219-01",
```

Comment:
Because this isn't the recommended way to specify the TraceID and TraceState I would drop these from the example.